tests/storage-buckets

```diff
@@ -22,6 +27,20 @@ if echo "${poolDriverList}" | grep -qwF "ceph"; then
   apt-get install --no-install-recommends --yes ceph-common
 fi
 
+# Fetch MinIO from its latest GitHub release tarball.
+minio_address="$(curl -s "https://api.github.com/repos/minio/minio/releases/latest" | jq '.tarball_url' | sed 's/"//g')"
+curl -L "${minio_address}" -o /tmp/minio.tar.gz
+
+# Clean up the build dir in case it hung around from a failed test.
+rm -rf /tmp/minio
+mkdir /tmp/minio
+
+# Build MinIO and apply it to the LXD snap.
+tar -xf /tmp/minio.tar.gz -C /tmp/minio
+make build -C /tmp/minio/*/
+mv /tmp/minio/*/minio /tmp/minio/
```

Comment:
Upstream provides builds for many arches according to https://min.io/download#/linux. If we can avoid building that'd be even better.
```
wget https://dl.min.io/server/minio/release/linux-amd64/minio
chmod +x minio
```

We can likely rely on `uname -m` with some mangling for `aarch64` and `ppc64el` mangling as usual but aside from that we should be good. They even provide `s390x`!

Reply:
Ah nice, very convenient. How come we were building it in the snap though?

Reply:
I don't know but maybe because we never used built artifacts except for those coming through Ubuntu packages? Also, it isn't very nice from a reproducible point of view to always pick their latest release but maybe they provide older versions somewhere too?

Reply:
Yeah, I guess if it always pulls the latest version then a `stable` snap would never actually be stable.

Reply:
Our GH pulls the prebuilt asset so for testing this is fine to do